core/src/main/java/org/opensearch/sql/expression/text/TextFunction.java

```diff
@@ -241,6 +242,18 @@ private DefaultFunctionResolver locate() {
                 TextFunction::exprLocate), INTEGER, STRING, STRING, INTEGER));
   }
 
+  /**
+   * Returns the position of the first occurrence of a substring in a string starting from 1.
+   * Returns 0 if substring is not in string.
+   * Returns NULL if any argument is NULL.
+   * Supports following signature:
+   * (STRING IN STRING) -> INTEGER
+   */
+  private DefaultFunctionResolver position() {
+    return define(BuiltinFunctionName.POSITION.getName(),
+            impl(nullMissingHandling(TextFunction::exprPosition), INTEGER, STRING, STRING));
```

Comment:
Why not use `TextFunction::exprLocate` here?

Makes `TextFunction::exprPosition` unnecessary.

Reply:
Addressed in 17bd91d4bf5ac4f6a3d7f95a949dd137a458c3df